src/library_dylink.js

```diff
@@ -449,7 +446,7 @@ var LibraryDylink = {
           }
           // otherwise this is regular function import - call it indirectly
           var resolved;
-          return obj[prop] = function() {
+          return function() {
```

Comment:
why disable the caching here?

Reply:
Because the object being proxied is now the asmLibraryArg.. I refactored to re-instate this cache mechanism.

This change is based on several other PR so actually is not quite ready for review yet.